extension/wrapper.cpp

```diff
@@ -95,9 +95,12 @@ static PyObject *
 Satrec_twoline2rv(PyTypeObject *cls, PyObject *args)
 {
     char *string1, *string2, line1[130], line2[130];
-    double dummy;
+    gravconsttype whichconst = wgs72;
+    double dummy;    
 
-    if (!PyArg_ParseTuple(args, "ss:twoline2rv", &string1, &string2))
+    if (!PyArg_ParseTuple(args, 
+            "ss|b:twoline2rv", 
```

Comment:
An enumerated type in C++ is the same storage size as a full integer even if it only has a few defined values, so `b` here needs to be `i` or this could fail on some architectures and C compilers.